src/settings/config.js

```diff
@@ -161,5 +173,23 @@ export default {
   ],
   footer: {
     copyright: 'Copyright \u00A9 2016 Iris'
+  },
+  petitionsStates: {
```

Comment:
Annoying but these shouldn't be in settings / over-writeable by the user. The need to be global constants elsewhere